src/main/java/no/unit/nva/cristin/projects/NvaProjectBuilder.java

```diff
@@ -46,19 +44,28 @@
     public static NvaProject mapCristinProjectToNvaProject(CristinProject cristinProject) {
```

Comment:
General comment: If you created a new NvaProjectBuilder with the cristinProject as a field the code would become more concise and with less parameters going back and fourth

Reply:
are you thinking `new NvaProjectBuilder(CristinProject cristinProject)` ?

Reply:
yes